js/src/services/panel.service.js

```diff
@@ -303,7 +303,7 @@ class PanelService {
         // clicking anywhere in main tag will close panel.
         $('main').on('click.atkPanel', debounce((evt) => {
             that.closePanel(id);
-        }, 100));
+        }, 250));
```

Comment:
Is it on purpose that 2 times 350 is used and 250 here?

Reply:
yes - 250 here is when user clicking outside panel and is not related to user typing something. Shorter value is ok.

Reply:
were there any server performance issues?

debounce feature slows tests - can we declare about the same debounce function and allow to override the debounce time for testing to like 5 ms

I already fixed all explicit waitings for AJAX requests, but this debounce is the last feature that need manual waiting now.

Reply:
Debounce is there for a reason. Please never downgrade user experience in order to satisfy test performance...

Reply:
> 
> 
> Debounce is there for a reason. Please never downgrade user experience in order to satisfy test performance...

I never do, I also want fast override for testing and also fast debounce for people that value every subsecond :)